src/apps/silo-finance/common/silo-finance.s-token.token-fetcher.ts

```diff
@@ -55,6 +55,14 @@ export abstract class SiloFinanceSTokenTokenFetcher extends AppTokenTemplatePosi
     const markets = await this.siloDefinitionResolver.getSiloDefinition(this.network);
     if (!markets) return [];
 
+    // console.log(markets);
+
+    const sTokens = markets.map(market => {
+      return market.marketAssets.map(marketAsset => { return marketAsset.sToken });
+    }).flat();
+
+    console.log(this.network, sTokens);
```

Comment:
Remove the console logs statements as a cleanup 🙏

Reply:
woopsies